docs/content/latest/explore/security/security.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Security
+headerTitle: Security
+linkTitle: Security
+description: Overview of security in YugabyteDB.
+headcontent: Security in YugabyteDB.
+image: /images/section_icons/index/secure.png
+aliases:
+  - /latest/explore/security/
+menu:
+  latest:
+    identifier: explore-security
+    parent: explore
+    weight: 930
+isTocNested: true
+showAsideToc: true
+---
+
+Like PostgreSQL, YugabyteDB provides security in multiple ways:
+
+* **Authentication** - limit access to the database to clients with proper credentials
+
+* **Authorization** - create users and roles, and grant privileges to restrict activities that the users and roles can perform
+
+* **Encryption** - encrypt the database, as well as all network communication between servers
+
+* **Auditing** - conduct session- and object-level auditing
+
+* **Restricting network access** - limit connections to the database using RPC binding
```

Comment:
If you want to have all your run-in headings parallel...
```suggestion
* **Network access restriction** - limit connections to the database using RPC binding
```

Reply:
fixed